dali/pipeline/operators/reader/video_reader_op.h

```diff
@@ -80,10 +80,11 @@ class VideoReader : public DataReader<GPUBackend, SequenceWrapper> {
       auto* sequence_output = tl_sequence_output.raw_mutable_tensor(data_idx);
 
       auto* prefetched_sequence = prefetched_batch_[data_idx];
-      CUDA_CALL(cudaMemcpy(sequence_output,
+      CUDA_CALL(cudaMemcpyAsync(sequence_output,
```

Comment:
maybe TypeInfo::Copy ?

Reply:
Thing is that I have to use it as that:
```
 tl_sequence_output.type().Copy<GPUBackend, GPUBackend>(sequence_output,
                                  prefetched_sequence->sequence.raw_data(),
                                  prefetched_sequence->sequence.size(),
                                  ws->stream());
```

Reply:
But I agree it's better since it allows us to have this part of this operator sort of CUDA agnostic

Reply:
Done